test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -69,5 +64,21 @@ describe('distribute with fractions', function () {
     ],
   ];
 
-  tests.forEach(t => testDistributeResult(t[0], t[1]));
+  const multiStepTests = [
+
+    ['(2 / x +  3x^2) * (x^3 + 1)',
+      ['(2 * (x^3 + 1) / x + 3x^2 * (x^3 + 1))',
+        '(2 * (x^3 + 1) / x + (3x^5 + 3x^2))']
+    ],
+
+    ['(2x + x^2) * (1 / (x^2 -4) + 4x^2)',
```

Comment:
ah okay, cool

there is a rule for that, but it's not used for distribution simplifications. maybe I'll just open another issue to add that in, since it's not really related to this PR - sound good?